src/encoded/static/components/navigation/components/SearchBar.js

```diff
@@ -214,19 +265,29 @@ export class SearchBar extends React.PureComponent{
 }
 
 const SelectItemTypeDropdownBtn = React.memo(function SelectItemTypeDropdownBtn(props){
-    const { currentAction, searchAllItems, toggleSearchAllItems, disabled = true } = props;
+    const { currentAction, searchItemType, onChangeSearchItemType, href, getDropdownTitleText, disabled = true } = props;
     if (isSelectAction(currentAction)) return null;
+
+    const showWithinResultsOption = useMemo(function(){
+        return SearchBar.isBrowseOrSearchPage(href);
+    }, [href]);
+
     return (
         <div className="search-item-type-wrapper">
             <DropdownButton id="search-item-type-selector" size="sm" variant="outline-secondary" disabled={disabled}
-                onSelect={(eventKey, evt)=>{ toggleSearchAllItems(eventKey === 'all' ? true : false); }}
-                title={searchAllItems ? 'All Items' : 'Experiment Sets'}>
-                <DropdownItem eventKey="sets" data-key="sets" active={!searchAllItems}>
-                    Experiment Sets
+                onSelect={(eventKey, evt) => { onChangeSearchItemType(eventKey); }}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JSX props should not use arrow functions (react/jsx-no-bind)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4513406)